components/slider/index.css

```diff
@@ -449,23 +483,247 @@ governing permissions and limitations under the License.
+  inset-block-start: 0;
 
-  margin-left: calc(var(--spectrum-slider-handle-width) / 2 * -1);
+  margin-inline-start: calc(var(--spectrum-slider-handle-size) / 2 * -1);
 }
 
 .spectrum-Slider-trackContainer {
-  height: var(--spectrum-slider-height);
+  block-size: var(--mod-slider-control-height, var(--spectrum-slider-control-height));
 
   /* stop edges from peeking out */
   overflow: hidden;
 }
+
+
+.spectrum-Slider-track {
+  &::before {
+    background: var(--highcontrast-slider-track-color, var(--mod-slider-track-color, var(--spectrum-slider-track-color)));
+  }
+}
+
+
+.spectrum-Slider-labelContainer {
+  color: var(--highcontrast-slider-label-text-color, var(--mod-slider-label-text-color, var(--spectrum-slider-label-text-color)));
+}
+
+
+.spectrum-Slider--filled {
+  .spectrum-Slider-track:first-child {
+    &::before {
+      background: var(--highcontrast-slider-track-fill-color, var(--mod-slider-track-fill-color, var(--spectrum-slider-track-fill-color)));
+    }
+  }
+}
+
+.spectrum-Slider-fill {
+  &::before {
+    background: var(--highcontrast-slider-track-fill-color, var(--mod-slider-track-fill-color, var(--spectrum-slider-track-fill-color)));
+  }
+}
+
+.spectrum-Slider-ramp {
+  path {
+    fill: var(--highcontrast-slider-ramp-track-color, var(--mod-slider-ramp-track-color, var(--spectrum-slider-ramp-track-color)));
+  }
+}
+
+.spectrum-Slider-handle {
+  border-color: var(--highcontrast-slider-handle-border-color, var(--mod-slider-handle-border-color, var(--spectrum-slider-handle-border-color)));
+  background: var(--highcontrast-slider-handle-background-color, var(--mod-slider-handle-background-color, var(--spectrum-slider-handle-background-color)));
+
+  &:hover {
+    border-color: var(--highcontrast-slider-handle-border-color-hover, var(--mod-slider-handle-border-color-hover, var(--spectrum-slider-handle-border-color-hover)));
+  }
+
+  &.is-focused {
+    border-color: var(--highcontrast-slider-handle-border-color-key-focus, var(--mod-slider-handle-border-color-key-focus, var(--spectrum-slider-handle-border-color-key-focus)));
+
+    &:before {
+      box-shadow: 0 0 0 var(--spectrum-focus-indicator-thickness) var(--highcontrast-slider-handle-focus-ring-color-key-focus, var(--mod-slider-handle-focus-ring-color-key-focus, var(--spectrum-slider-handle-focus-ring-color-key-focus)));
+    }
+  }
+
+  &:active,
+  &.is-dragged {
+    border-color: var(--highcontrast-slider-handle-border-color-down, var(--mod-slider-handle-border-color-down, var(--spectrum-slider-handle-border-color-down)));
+  }
+}
+
+
+.spectrum-Slider--ramp {
+  .spectrum-Slider-handle {
+    box-shadow: 0 0 0 var(--spectrum-slider-handle-gap) var(--highcontrast-slider-ramp-handle-border-color-active, var(--mod-sectrum-slider-ramp-handle-border-color-active, var(--spectrum-slider-ramp-handle-border-color-active)));
+
+    /* TODO: currently in tokens v5.0.0 the background-color for the handle are not yet defined */
```

Comment:
I'm not sure I understand the comment - is this still accurate? We're on @adobe/spectrum-tokens v12.8.0 now and our package is at v10

Reply:
I don't think it's accurate any longer. I picked this work up in the middle but there are tokens assigned to this value obviously so I'll remove the comment